### PR TITLE
18194: DIG063 - No Binder Payment Received

### PR DESCRIPTION
**Pull request by GouthamRedd**
|Field|Value|
|-----|-----|
| Redmine Ticket Number | [18194](https://devops.dchbx.org/redmine/issues/18194) |
| App-Dev Road Map # | n/a |
| Start Date | n/a |
| Due Date | n/a  |
| App-Dev Priority | UP-3 |
| Development Story Points | n/a |
| Peer Review Story Points | 0.5 |
| Ticket Author | Chloe Berger |

### CODE REVIEW

File: app/models/employer_profile.rb [hunk 1]
```diff
@@ -971,6 +985,13 @@ def enrollment_ineligible_period_expired?
   # def is_eligible_to_shop?
   #   registered? or published_plan_year.enrolling?
   # end
+  def self.initial_employers_enrolled_plan_year_state
```
Comment (Battula) [inline app/models/employer_profile.rb]: Use start_on to filter employers

File: app/views/notices/shop_employer_notices/notice_to_employer_no_binder_payment_received.html.erb [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    </tr>
+  </tbody>
+</table>
+
+<%= render partial: "notices/shared/date", :locals => {notice: notice} %>
+
+<table class="total_table" border="0">
+  <tbody>
+    <tr>
+      <td><h3>No Payment for Group Health Coverage on DC Health Link</h3></td>
```
Comment (Battula) [inline app/views/notices/shop_employer_notices/notice_to_employer_no_binder_payment_received.html.erb]: Use Settings

File: config/settings.yml [hunk 3]
```diff
@@ -7,6 +7,7 @@ site:
   logo_file_name: "logo.png"
   home_url: "https://www.dchealthlink.com/"
   website_name: "www.DCHealthLink.com"
+  employer_application_deadline_link: "https://www.dchealthlink.com/smallbusiness/employer-coverage-deadlines"
```
Comment (Battula) [inline config/settings.yml]: Can we move this to aca.shop_market

File: app/models/employer_profile.rb (hunk 1 above)
Comment (GouthamRedd) [reply app/models/employer_profile.rb]: Used start_on for the initial_employer_enrooled_plan_year_state to filter employers.

File: app/views/notices/shop_employer_notices/notice_to_employer_no_binder_payment_received.html.erb (hunk 2 above)
Comment (GouthamRedd) [reply app/views/notices/shop_employer_notices/notice_to_employer_no_binder_payment_received.html.erb]: Called the DC Health Link by using settings.

File: config/settings.yml (hunk 3 above)
Comment (GouthamRedd) [reply config/settings.yml]: yes moved employer_application_deadline_link to aca.shop_market.

**Pull request closed without merging**